.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+## [BUG REPORT TEMPLATE]
```

Comment:
It seems to me that the template is well suited for reporting software issues, but given how this is mostly about metrics definitions, maybe it would be better to have template more oriented to that (or even no template at all).

Reply:
I feel that the general division of issue reports into two categories:

1. "something that's here is broken" (i.e. a bug report)
2. "something is not here that should be" (i.e. an enhancement request)

works well for more than just software.  

I'm happy to consider concrete proposals to change wording  / layout etc., of course.  What do you have in mind, @jgbarah ?

Reply:
I was just referring to stuff like "Steps to reproduce" or "Environment" (with references about OS version, for example). Maybe something more simple, along the lines you propose (is this a bug report or an enhacement report), and details prompting for the link to the document and specific lines, or something like that... For example, for bugs:

```
[BUG REPORT TEMPLATE]

### Description of the bug

Detailed description of the part of the document(s) which are buggy, and why.

### Links

Include references to the documents and lines in those documents, when convenient.

### Proposal for fixing

If you can, propose how to fix the bug.
```

Something similar for enhacements (or just the same template, with an option of labeling it as "bug report" or "enhacement".

What do you think?

Reply:
LGTM!

For enhancements, I prefer problem-focused statements rather than solution-focused statements.  Recommending or requiring user story / job story format can help to ensure that.